IssueComment
  { issueCommentUpdatedAt = 2015 (-10) (-21) 00 : 41 : 56 UTC
  , issueCommentUser =
      SimpleUser
        { simpleUserId = Id 552910
        , simpleUserLogin = N "Tryneus"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/552910?v=3"
        , simpleUserUrl = "https://api.github.com/users/Tryneus"
        , simpleUserType = OwnerUser
        }
  , issueCommentUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/comments/149744668"
  , issueCommentHtmlUrl =
      "https://github.com/rethinkdb/rethinkdb/issues/4973#issuecomment-149744668"
  , issueCommentCreatedAt = 2015 (-10) (-21) 00 : 41 : 56 UTC
  , issueCommentBody =
      "Note that in release mode, this will just print an error and continue on with no problem - the table availability issue will just have an `__unknown_server__` value in it."
  , issueCommentId = 149744668
  }